Elysium/build.gradle

```diff
@@ -1,34 +1,42 @@
 plugins {
     id 'java'
+    id 'org.springframework.boot' version '3.0.3'
+    id 'io.spring.dependency-management' version '1.1.0'
 }
 
-group 'com.example'
-version '1.0-SNAPSHOT'
+group = 'hse.elysium'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '17'
 
 repositories {
     mavenCentral()
 }
-
 ext {
     junitVersion = '5.9.1'
```

Comment:
Versions can be stored in gradle.properties

Reply:
internet says ext is also a good place to store